packages/web-app-files/src/App.vue

```diff
@@ -52,26 +49,15 @@ export default {
     }
   },
   computed: {
-    ...mapGetters('Files', ['dropzone', 'highlightedFile', 'inProgress']),
-
-    _sidebarOpen() {
-      return this.highlightedFile !== null
-    },
+    ...mapGetters('Files', ['dropzone', 'inProgress']),
 
     $_uploadProgressVisible() {
       return this.inProgress.length > 0
     }
   },
   watch: {
     $route() {
-      this.$_destroySideBar_hideDetails()
       this.resetFileSelection()
-    },
-    highlightedFile(file) {
-      if (file !== null) {
-        return
-      }
-      this.$_destroySideBar_hideDetails()
```

Comment:
Removing this line removes a little bit more logic... There is a mixin (`destroySideBar.js`) which takes care of resetting the active sidebar panel. Perhaps better solved within the sidebar though. Let's discuss it later today.